### PR TITLE
Changed call controller layout

### PR DESCRIPTION
**Pull request by Jaseemakhtar**
### Things to look out for:

- Added some drawables.
- Made some changes to design of call controller layout i,e. webview_fragment.xml

### CODE REVIEW

File: app/src/main/res/layout/webview_fragment.xml [hunk 1]
```diff
@@ -71,7 +71,7 @@
         android:textColor="@android:color/white"
         android:textStyle="bold"
         android:id="@+id/bannerCallActionNotice"
-        android:background="@color/colorPrimaryDark"
```
Comment (tomriddle25) [inline app/src/main/res/layout/webview_fragment.xml:74]: Don't use a TextView as the background for the bottombar. Use something like a plain `View`. It has no use being a textview.

Comment (shahsurajk) [reply app/src/main/res/layout/webview_fragment.xml:74]: That's my code. :speak_no_evil: 

Originally had plans to show some text there.

Comment (tomriddle25) [reply app/src/main/res/layout/webview_fragment.xml:74]: @shahsurajk it's okay. You'll learn with time. 🙈

Comment (tomriddle25) [reply app/src/main/res/layout/webview_fragment.xml:74]: @Jaseemakhtar Make the change, push and request PR.

Comment (Jaseemakhtar) [reply app/src/main/res/layout/webview_fragment.xml:74]: Okay.

**Review by tomriddle25 — APPROVED** [review]
Cool.

**Pull request merged**